Testing/VelaptorTests/Content/ContentLoaderTests.cs

```diff
@@ -87,92 +81,88 @@ public void Load_WhenLoadingFonts_LoadsFont()
             var loader = CreateContentLoader();
 
```

Comment:
Refactor method to match unit test
```suggestion
public void LoadFont_WhenLoadingFonts_LoadsFont()
```